delete-all-nodes/README.md

```diff
@@ -1,6 +1,6 @@
-# Drain a Kubernetes Node Chaos Experiment
+# Delete Kubernetes Nodes Chaos Experiment
 
-This experiment explores whether my application can maintain itself when a Kubernetes node is drained
+This experiment explores whether my application when all of a Kubernetes cluster nodes are deleted. The implication here is you have multiple clusters for your application and it will failover to another cluster.
```

Comment:
Suggest changing to:

"The belief being explored here is that you have multiple clusters..."